tests/unit/reporter.js

```diff
@@ -0,0 +1,46 @@
+const { readFileSync, writeFileSync } = require('fs');
+const path = require('path');
+const { summarizeData } = require('./utils');
+
+class Reporter {
+  constructor(globalConfig, options) {
+    this._globalConfig = globalConfig;
+    this._options = options;
+  }
+
+  onRunComplete() {
+    // if the coverage flag is false, no need to parse json summary documents
+    if (this._globalConfig.collectCoverage !== true) return;
+
+    const coveragePath = path.resolve(__dirname, 'coverage-data', 'coverage-summary.json');
+    const coverageReport = readFileSync(coveragePath, 'utf-8');
```

Comment:
Should there be some error handling for `readFileSync` to handle cases where the file might not exist or be unreadable?

Reply:
Yep! Added here: https://github.com/CMSgov/design-system/pull/3359/commits/cce952c28836fbee69714bd638bc5059ac4b2b31